tools/cc/eosio-cpp.cpp.in

```diff
@@ -11,6 +11,8 @@
 #include "clang/Rewrite/Core/Rewriter.h"
 #include "clang/Rewrite/Frontend/Rewriters.h"
 #include "llvm/Support/FileSystem.h"
+#include "clang/Lex/Preprocessor.h"
```

Comment:
These two unused headers can be removed.

Reply:
done